dbt/adapters/bigquery/utility.py

```diff
@@ -43,3 +45,39 @@ def sql_escape(string):
     if not isinstance(string, str):
         raise dbt_common.exceptions.CompilationError(f"cannot escape a non-string: {string}")
     return json.dumps(string)[1:-1]
+
+
+def is_base64(s: str | bytes) -> bool:
```

Comment:
Failing test:
```
dbt/adapters/bigquery/utility.py:50: error: X | Y syntax for unions requires Python 3.10  [syntax]
```
Could you make this a `Union` instead? We still support python 3.9, and this syntax was only introduced in python 3.10

Reply:
Done! It slipped my mind that dbt is 3.9+